pkg/resource/resource.go

```diff
@@ -142,9 +142,14 @@ func ResolveClassClaimValues(classValue, claimValue string) (string, error) {
 }
 
 // SetBindable indicates that the supplied Bindable is ready for binding to
-// another Bindable, such as a resource claim or managed resource.
+// another Bindable, such as a resource claim or managed resource by setting its
+// binding phase to "Unbound". It is a no-op for Bindables in phases "Bound" or
+// "Released", because these phases may not transition back to "Unbound".
 func SetBindable(b Bindable) {
-	if b.GetBindingPhase() == v1alpha1.BindingPhaseBound {
+	switch b.GetBindingPhase() {
```

Comment:
Using `switch` instead of just an `OR` operator feels odd to me here.

Reply:
I don't feel strongly either way on this one. I tend to prefer switches because I think line wrapped if statements are ugly, and it's easy for `||` if statements to get long in Go.

Reply:
I force pushed a change that attempts to resolve this by doubling down on the switch statement. ;)